syncd/FlexCounter.cpp

```diff
@@ -1597,7 +1597,7 @@ void FlexCounter::runPlugins(
     {
         std::to_string(counters_db.getDbId()),
         COUNTERS_TABLE,
-        std::to_string(m_pollInterval * 1000)
+        std::to_string(m_pollInterval / 1000)
```

Comment:
m_pollInterval is set via orchagent with values in milliseconds (MS), currently this value was extended to microseconds, and now you changing this to seconds, so if m_pollInterval will be equal to let say 60ms or 100ms or 900ms, then your value in db will be set to zero (0) because of integer division, is that what you want ?

Reply:
Done - removed the multiplication and did the required multiplication in the lua script - https://github.com/Azure/sonic-swss/pull/1855